terraform/cloudsql/mysql/bind/outputs.tf

```diff
@@ -15,12 +15,13 @@ output "uri" {
 output "port" { value = local.port }
 output "jdbcUrl" {
   sensitive = true
-  value = format("jdbc:mysql://%s:%d/%s?user=%s\u0026password=%s\u0026useSsl=true\u0026disableSslHostnameVerification=true",
+  value = format("jdbc:mysql://%s:%d/%s?user=%s\u0026password=%s\u0026useSsl=%s\u0026disableSslHostnameVerification=true",
```

Comment:
question: should the `disableSslHostnameVerification` option added only when `!var.allow_insecure_connections`?

Reply:
Good point